x-pack/plugins/security_solution/server/lib/timeline/routes/timelines/get_timeline/index.ts

```diff
@@ -42,34 +39,18 @@ export const getTimelineRoute = (
         const frameworkRequest = await buildFrameworkRequest(context, security, request);
         const query = request.query ?? {};
         const { template_timeline_id: templateTimelineId, id } = query;
+
         let res = null;
+
         if (templateTimelineId != null && id == null) {
           res = await getTimelineTemplateOrNull(frameworkRequest, templateTimelineId);
         } else if (templateTimelineId == null && id != null) {
           res = await getTimelineOrNull(frameworkRequest, id);
-        } else if (templateTimelineId == null && id == null) {
```

Comment:
you removed that code because it can NOT happen with our UI as it is today

Reply:
That case was for retrieving all timelines when no template timeline id or timeline id is given. I move that part of logic to x-pack/plugins/security_solution/server/lib/timeline/routes/timelines/get_timelines/index.ts